### ISSUE TITLE
Broken docker image uploaded recently

### ISSUE THREAD

**Issue by dacarley** [mvertes/docker-alpine-mongo#32]
The most recent image of `mvertes/alpine-mongo` appears to contain Mongo 3.6.7, not Mongo 4.0.6, as expected.

![image](https://user-images.githubusercontent.com/963715/55255500-09e1da80-5218-11e9-8ed6-8d252ae0d162.png)

For comparison, here's with the `4.0.5-0` tag:

![image](https://user-images.githubusercontent.com/963715/55255532-2251f500-5218-11e9-9215-5d6a73ace8eb.png)

**Comment by mvertes** [comment]
Ok, thanks for your report. It appears that the image is built from alpine:3.8, which contains mongo-3.6.7, instead of alpine:edge, which contains the last 4.0.6.
Fixing it.

**Comment by Svetomechc** [comment]
https://pkgs.alpinelinux.org/packages?name=mongodb&branch=edge
The package appears to be gone from edge repositories for good... any idea why? 

Is there a new way to install latest version of MongoDB in Alpine?

**Comment by russkel** [comment]
@Svetomechc It has been moved to the nonfree repository https://github.com/alpinelinux/aports/tree/master/non-free/mongodb

**Comment by Svetomechc** [comment]
@russkel ok, thanks for the info! Never installed from there, though. Is there a guide?

**Comment by russkel** [comment]
@Svetomechc likewise. I believe you edit /etc/apk/repositories make a copy of one of the lines and s/community/nonfree/ ;)
Let me know how you go.

**Comment by Svetomechc** [comment]
@russkel I just checked that the version there is actually the same I have already - **4.0.6** (guess I downloaded it right before the takedown). So, to use the latest **4.0.10** I will probably have to download one of the [official images](https://hub.docker.com/_/mongo) based on Ubuntu, not Alpine. Sad.

**Comment by russkel** [comment]
@Svetomechc I was looking at updating the package but it has so many bloody patches. I also can't get mongo-cxx-driver to compile at the moment. It's all a bit of a mess, which is surprising given the perceived popularity of MongoDB.

**Comment by Svetomechc** [comment]
@russkel yeah, was really surprised too. And here's the radio silence when it went away from Alpine repos in our case. Literally no google results about this happening, you have to find out about this manually.

**Comment by russkel** [comment]
@Svetomechc I have had success in building the drivers. I will probably upstream the packages to aports, if that's any help to you.

**Comment by Svetomechc** [comment]
@russkel let me know if you do. Though didn't find any documentation on [their site](https://git.alpinelinux.org/aports/about/) on how to use them.

**Comment by russkel** [comment]
@Svetomechc I am in the process of getting the drivers on edge testing, see https://github.com/alpinelinux/aports/pull/9244

As for using the testing repository, please see https://wiki.alpinelinux.org/wiki/Edge

**Comment by Svetomechc** [comment]
@russkel only drivers, not mongodb itself?
And I use edge repositories already like so: `echo "http://mirror.yandex.ru/mirrors/alpine/edge/testing" >> /etc/apk/repositories`, but weren't we talking about aports, or is it the same thing?

**Comment by russkel** [comment]
@Svetomechc at this point yes. I will take a shot at the db itself at some point.

Yes that's all you need. Aports are the scripts used to build the packages in the repositories.
